rest-api/nvswitch-manager/README.md

```
@@ -73,35 +78,61 @@ go build -o nvswitch-manager
 ```
 
 ### 4. Start the NV-Switch Manager gRPC service
-Run with a persistent backend (Postgres + Vault):
 
+For local work, run in-memory — no database, no NICo Core, no certificates:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)